test/nidmExampleDataTest.m

```diff
@@ -8,38 +8,35 @@
     methods(TestMethodSetup)
         %Rename the users HTML folder to prevent the tests damaging it.
         function storeUsersHTML(testCase)
-            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7 
-                movefile(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html'), strrep(fileparts(mfilename('fullpath')), 'test', 'Data\htmlTemp'))
+            if exist(fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'html'), 'dir') == 7 
+                movefile(fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'html'), fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'htmlTemp'))
             end
         end
     end
     
     methods(TestMethodTeardown)
-        %Remove the HTML folder created by test.
+        %Remove the HTML folder created by test and move the users data back to the HTML folder.
         function removeTestHTML(testCase)
-            pause(0.1);
-            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7
-                rmdir(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html'), 's')
+            com.mathworks.mlservices.MatlabDesktopServices.getDesktop.closeGroup('Web Browser');
```

Comment:
I have managed to run the tests on my machine and they both pass but I get the following error:
```
Totals:
   2 Passed, 0 Failed, 0 Incomplete.
   2.7279 seconds testing time.

Exception in thread "AWT-EventQueue-0" java.lang.IllegalStateException: Channel is already closed.
	at com.teamdev.jxbrowser.chromium.internal.ipc.SocketChannelWriter.write(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.SocketChannel.send(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.a(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.loadURL(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.loadURL(Unknown Source)
	at com.mathworks.html.jxbrowser.chromium.JxBrowserChromiumNavigator$1.run(JxBrowserChromiumNavigator.java:119)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
	at java.awt.EventQueue.access$200(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:694)
	at java.awt.EventQueue$3.run(EventQueue.java:692)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
Exception in thread "AWT-EventQueue-0" java.lang.IllegalStateException: Channel is already closed.
	at com.teamdev.jxbrowser.chromium.internal.ipc.SocketChannelWriter.write(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.SocketChannel.send(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.a(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.loadURL(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.WebView.loadURL(Unknown Source)
	at com.mathworks.html.jxbrowser.chromium.JxBrowserChromiumNavigator$1.run(JxBrowserChromiumNavigator.java:119)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
	at java.awt.EventQueue.access$200(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:694)
	at java.awt.EventQueue$3.run(EventQueue.java:692)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
```

To solve the problem, I wonder if you could replace line 20 by:
```
[stat,h] = web()
close(h)
```
or even better update `spm_results_export.m` to return the handle:
```
[stat, h] = web(fHTML);
```
and in turn return it from `nidm_results_display` so that the window can be closed with `close(h)` later on. 

What do you think of this suggestion?